props/prop_base.h

```diff
@@ -1571,11 +1571,15 @@ class PropBase : CommandParser, protected Looper, protected SaberBase, public Mo
       return true;
     }
     if (!strcmp(cmd, "n") || (!strcmp(cmd, "next") && arg && (!strcmp(arg, "preset") || !strcmp(arg, "pre")))) {
-      next_preset();
+      SaberBase::IsOn() ? next_preset_fast() : next_preset();
       return true;
     }
     if (!strcmp(cmd, "p") || (!strcmp(cmd, "prev") && arg && (!strcmp(arg, "preset") || !strcmp(arg, "pre")))) {
-      previous_preset();
+      SaberBase::IsOn() ? previous_preset_fast() : previous_preset();
+      return true;
+    }
+    if (!strcmp(cmd, "f") || (!strcmp(cmd, "first") && arg && (!strcmp(arg, "preset") || !strcmp(arg, "pre")))) {
```

Comment:
It feels like "first preset" already exists as "set_preset 0".

Reply:
Fair. I forgot about set_preset.
Removed.